runner.py

```diff
@@ -0,0 +1,495 @@
+#!/usr/bin/env python
+
+import sys
+assert sys.version_info >= (3, 9), "Use Python 3.9 or newer."
+
+import os
+import socket
+import optparse
+import datetime
+import shutil
+import subprocess
+import time
+import signal
+import json
+from typing import Union, Tuple
+
+'''
+The default flags depending on the hostname.
+'''
+DEFAULT_SETTINGS = {
+    'futharkhpa01fl.unicph.domain': {
+        'name': 'A100',
```

Comment:
This one should be called `futhark01`.  (We have multiple machines with A100s.)

Reply:
This should be fixed in the latest's commit.